source/commands/utils/sharedDangerfileArgs.ts

```diff
@@ -38,3 +40,4 @@ export default (command: any) =>
     .option("-c, --external-ci-provider [modulePath]", "Specify custom CI provider")
     .option("-p, --process [command]", "[dev] Runs a custom sub-process instead of the Danger JS runtime")
     .option("-u, --passURLForDSL", "[dev] Use a custom URL to send the Danger DSL into the sub-process")
+    .option("--use-github-checks", "Use GitHub Checks", false)
```

Comment:
Whaddayareckon?
I didn't dare try to come up with a single-character option for this.